doc/en/components/charts/chart-types-area.md

```diff
@@ -22,6 +26,8 @@ mentionedTypes: ["XamCategoryChart", "XamDataChart"]
 
 ## $PlatformShort$ Area Chart with Multiple Series
 
+Since the $PlatformShort$ Area Chart allows you to combine multiple series and compare or see how they change over time. All we need to do is bind to a data source containing the data for China and the USA, and the area chart will automatically update to fit the additional data.
+
 <div class="sample-container loading" style="height: 400px">
```

Comment:
this sentence doesn't make sense - starting with Since

Reply:
Removed "Since the"